packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -72,17 +73,19 @@ class DateAbstract extends React.Component {
     return null
   }
 
-  componentWillReceiveProps(props) {
-    this.handleButtonVisibility()
-    const locale = this.getLocale(props.intl.locale)
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (!_isEqual(prevProps.value, this.props.value)) {
+      this.handleButtonVisibility()
+      const locale = this.getLocale(this.props.intl.locale)
```

Comment:
so if `this.props.intl.locale ` changes this will not update since you wrapped everything in a "only when value changed"-if?